pkg/common/moerr/internal_error_define.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package moerr
+
+var (
+	ErrNoHardwareAddr     = NewInternalError("uuid: no Hardware address found")
```

Comment:
Why do this?

Should just be NewNoHardwareAddr, or, just use NewInternalError

Reply:
Fixed